src/runtime_src/core/include/xcl_api_macros.h

```diff
@@ -26,7 +26,8 @@ mtx.lock(); \
 mtx.unlock();
 
 #define RPC_PROLOGUE(func_name) \
-    unix_socket* _s_inst = sock; \
+    /*unix_socket* _s_inst = sock; */ \
```

Comment:
Please remove the commented code. We can refer in history of the file.

Reply:
Agreed. Removed commented Code.